demo/src/actions.ts

```diff
@@ -1,42 +1,59 @@
-function throwError() {
+import '@grafana/agent-web/dist/globals';
```

Comment:
I wanted extending a global type to be explicit choice rather than happening automatically by importing from a barrel. 
Two reasons:
* It might be wrong if you renamed the global from "grafanaAgent" via config
* Extending global types can be contraversial